src/lib/pages/backoffice/Loan/LoanSearch/SearchDateRange.js

```diff
@@ -20,17 +19,10 @@ export class _SearchDateRange extends Component {
     return [fromDate, toDate];
   }
 
-  /** react-searchkit allows having the same filter multiple times with
-   * different values. For this range dates filters, we want each filter to
-   * appear only one time with one value (e.g. loan_start_date = `<date>`)
-   */
   onDateChange = (newFilter) => {
     const { currentQueryState, updateQueryState } = this.props;
-    const [name, value] = newFilter;
-    const filters = currentQueryState.filters.filter(
-      (filter) => filter[0] === name
-    );
-    if (!_isEmpty(value)) filters.push(newFilter);
+    const filters = currentQueryState.filters;
+    filters.push(newFilter);
```

Comment:
This was tricky when we implemented it long time ago, I hope you were able to test any possible scenario (only one date input, or both, or remove them)

Reply:
I tested it, also with a combination of other facets and it seemed to work fine.